spec/controllers/miq_policy_controller/rsop_spec.rb

```diff
@@ -0,0 +1,28 @@
+include UiConstants
```

Comment:
@Fryguy @chessbyte It appears that there are ~20 specs that include this module globally as is done here, unfortunately.

These should be removed and injected via the [spec configuration](https://github.com/manageiq/manageiq/blob/10929f3bd76032264cba23f1e4c71bcb78276bd4/spec/spec_helper.rb#L48-L51) if they are really required for testing at all (arguably, that might not be true).